worker/src/main/scala/org/apache/celeborn/service/deploy/worker/FetchHandler.scala

```diff
@@ -198,6 +203,16 @@ class FetchHandler(val conf: CelebornConf, val transportConf: TransportConf)
           logDebug(s"Received chunk fetch request $shuffleKey $fileName $startIndex " +
             s"$endIndex get file info $fileInfo from client channel " +
             s"${NettyUtils.getRemoteAddress(client.getChannel)}")
+          if (readLocalShuffle) {
+            return replyStreamHandler(
```

Comment:
It's a bit strange to return `replyStreamHandler` because it returns `Unit`. IMO it's better to remove the `return`, and change L216's `if` to `else`.